src/commands/auth.ts

```diff
@@ -107,6 +124,28 @@ that configuration file swiftly.`)
       return
     }
 
+    if (system) {
```

Comment:
If  a value for the "system" flag is provided but the config file has not been upgraded yet, you get the following:

![CleanShot 2023-07-12 at 21 16 31](https://github.com/nuance-communications/mix-cli/assets/627065/59f2a84c-55b6-47a3-a9e6-bd4eb477971f)

which is confusing. Granted, this should not happen but users may end up typing the "auth" command with a system flag without having their config upgraded first. 

Can we upgrade the config file first when running the "auth" command (with or without the "system" flag)?